overlord/servicestate/servicestate.go

```diff
@@ -317,7 +319,16 @@ func (sd *StatusDecorator) DecorateWithStatus(appInfo *client.AppInfo, snapApp *
 // SnapServiceOptions computes the options to configure services for
 // the given snap. This function might not check for the existence
 // of instanceName.
-func SnapServiceOptions(st *state.State, instanceName string) (opts *wrappers.SnapServiceOptions, err error) {
+func SnapServiceOptions(st *state.State, quotaGroups map[string]*quota.Group, instanceName string) (opts *wrappers.SnapServiceOptions, err error) {
```

Comment:
nitpick: I would  have quotaGroups be the last argument, especially also given it can take a nil

Reply:
also the doc comment should say something about it

Reply:
done